### PR TITLE
acl: Test basic ACL IR and STORAGE rules compliance

### PR DESCRIPTION
**Pull request by vvarg229**
Added a test to verify compliance rules with the basic ACL IR and STORAGE. This is a copy of a similar test from eACL.

Closes https://github.com/nspcc-dev/neofs-testcases/issues/634

### CODE REVIEW

File: pytest_tests/testsuites/acl/test_acl.py [hunk 1]
```diff
@@ -178,3 +180,149 @@ def test_basic_acl_readonly(self, wallets, client_shell, read_only_container, fi
...
+
+        cid = public_container
+
+        with allure.step("Add test objects to container"):
+            owner_object_oid = put_object_to_random_node(
+                user_wallet.wallet_path, file_path, cid, shell=self.shell, cluster=self.cluster
+            )
+
+        with allure.step("Check IR and STORAGE rules compliance"):
+            assert not can_put_object(
```
Comment (evgeniiz321) [inline pytest_tests/testsuites/acl/test_acl.py:203]: Can we use ```check_full_access_to_container (robot/resources/lib/python_keywords/container_access.py)``` or ```check_no_access_to_container``` here? or some modification of it maybe?

File: pytest_tests/testsuites/acl/test_acl.py [hunk 2]
```diff
@@ -178,3 +180,149 @@ def test_basic_acl_readonly(self, wallets, client_shell, read_only_container, fi
...
+                storage_wallet.wallet_path,
+                cid,
+                shell=self.shell,
+                endpoint=endpoint,
+                oid=owner_object_oid,
+                wallet_config=storage_wallet.config_path,
+            )
+
+            with pytest.raises(AssertionError):
+                assert can_get_range_of_object(
```
Comment (evgeniiz321) [inline pytest_tests/testsuites/acl/test_acl.py:274]: AssertionError is expected to be raised from the assert here? Or from somewhere internally? If from this assert, maybe better to use just ```assert no can_get...```

Comment (evgeniiz321) [reply pytest_tests/testsuites/acl/test_acl.py:274]: Also it is not clear from the test why some operations are allowed and others not. It is better to add some related comment

Comment (vvarg229) [reply pytest_tests/testsuites/acl/test_acl.py]: > AssertionError is expected to be raised from the assert here? Or from somewhere internally? If from this assert, maybe better to use just `assert not can_get...`

Fixed

Comment (vvarg229) [reply pytest_tests/testsuites/acl/test_acl.py]: > Also it is not clear from the test why some operations are allowed and others not. It is better to add some related comment

Fixed

File: pytest_tests/testsuites/acl/test_acl.py (hunk 1 above)
Comment (evgeniiz321) [reply pytest_tests/testsuites/acl/test_acl.py]: They've already been put there. ```check_full_access_to_container``` looks pretty much the same as these conditions here with some variations. But ok, we can leave it as is

File: pytest_tests/testsuites/acl/test_acl.py [hunk 3]
```diff
@@ -178,3 +180,180 @@ def test_basic_acl_readonly(self, wallets, client_shell, read_only_container, fi
                 shell=client_shell,
                 cluster=self.cluster,
             )
+
+    @allure.title("Test basic ACL IR and STORAGE rules compliance")
+    def test_basic_acl_ir_storage_rules_compliance(self, wallets, public_container, file_path):
```
Comment (evgeniiz321) [inline pytest_tests/testsuites/acl/test_acl.py:185]: No type annotations here, at least some can be added

File: pytest_tests/testsuites/acl/test_acl.py (hunk 1 above)
Comment (vvarg229) [reply pytest_tests/testsuites/acl/test_acl.py]: > looks pretty much the same

But it's not full access here

File: pytest_tests/testsuites/acl/test_acl.py (hunk 3 above)
Comment (vvarg229) [reply pytest_tests/testsuites/acl/test_acl.py]: > No type annotations here, at least some can be added

Thank you very much! Sorry, I wasn't paying attention.
Fixed.

**Review by evgeniiz321 — APPROVED** [review]

**Pull request merged**